### DISCUSSION TITLE
Fast RTMP data input to OvenMediaEngine

### DISCUSSION THREAD

**Discussion by wzelek** [OvenMediaLabs/OvenMediaEngine#1523] [Help]
Hello, 
I'm wondering if Oven Media Engine supports fast replaying of RTMP data. 

Trying to inject using Gstreamer an RTMP stream (sync=False) with an 1h audio very fast, and then produce LLHLS output via OME. 

When replaying slowly everytjong is ok, HLS output is not truncated, for 1s chunks im getting 3600 HLS files. 
with fast replay im getting wrong output, truncated at the begginging or end(its random) (1700 HLS files created)

**Comment by basisbit** [comment]
What log output do you get? I assume that there is log output complaining about input buffer overflowing
